src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -1014,6 +1014,27 @@ def get_destination_region(destination):
         dest_project_id = dxpy.WORKSPACE_ID
     return dxpy.api.project_describe(dest_project_id, input_params={"fields": {"region": True}})["region"]
 
+def verify_nf_license(destination, extra_args):
+    
+    def get_project_to_check():
+        # extra args overrides the destination argument
+        # so we're checking it first
+        if "project" in extra_args:
```

Comment:
corner case here: what if in extra_args we have `destination` instead?

Reply:
It will be ignored since it's not a valid input to [/applet/new](https://documentation.dnanexus.com/developer/api/running-analyses/applets-and-entry-points#api-method-applet-new).

Reply:
Then just curious: what if the billTo is also overrided using `--bill-to`? do we need to check it?

Reply:
This is an option only applicable to apps :)